src/ContainerDetails.jsx

```diff
@@ -25,6 +33,8 @@ const ContainerDetails = ({ container }) => (
             <dd>{container.command ? container.command.join(" ") : ""}</dd>
             <dt>{_("State")}</dt>
             <dd>{render_container_state(container)}</dd>
+            <dt>{_("Ports")}</dt>
+            <dd>{render_container_published_ports(container)}</dd>
```

Comment:
Do we want to hide those when there are no published ports?

Reply:
Hm well we leave fields empty if they're not filled, so I went with that. 

But we can start doing this of course.

Reply:
Yeah sorry I meant to ping @garrett for a design perspective.

Reply:
I also think it makes more sense to show an empty input by default than an extra checkbox that  will show/hide them, that feels too clumsy.

Reply:
I removed the checkbox, I'll upload new screenshots